### PR TITLE
Update deploy-and-configure-the-report-message-add-in.md

### LINKED ISSUE #13085: Outdated information
The information on this page is not accurate now that the 365 defender pages have been moved around.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 1541fb66-1414-579d-344c-fc1e25ba4ea0
* Version Independent ID: 1541fb66-1414-579d-344c-fc1e25ba4ea0
* Content: [How-to deploy and configure the report message add-in](https://learn.microsoft.com/en-us/microsoft-365/security/office-365-security/step-by-step-guides/deploy-and-configure-the-report-message-add-in?view=o365-worldwide)
* Content Source: [microsoft-365/security/office-365-security/step-by-step-guides/deploy-and-configure-the-report-message-add-in.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/step-by-step-guides/deploy-and-configure-the-report-message-add-in.md)
* Service: **microsoft-365-security**
* Sub-service: **mdo**
* GitHub Login: @MSFTBen
* Microsoft Alias: **tracyp**

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the steps in configure add-in for users section

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/13085

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/office-365-security/step-by-step-guides/deploy-and-configure-the-report-message-add-in.md [hunk 1]
```diff
@@ -57,10 +57,8 @@ Depending on whether you are licensed for Defender for Office 365, you'll also g
 1. **Login** to the Microsoft Security portal at <https://security.microsoft.com>.
 2. On the left nav, select **Settings** and choose **Email & collaboration**.
 3. Select **User reported settings**.
-4. Ensure **Microsoft Outlook Report Message button** is toggled to **On**.
-5. Under **Send the reported messages to** choose **Microsoft** (Recommended).
-6. Ensure **Let users choose if they want to report** is unchecked and **Always report the message** is selected.
-7. Press **Save**.
+4. Ensure **Monitor report messages in outlook** is selected and select **use the built-in report button**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/step-by-step-guides/deploy-and-configure-the-report-message-add-in.md]:
```suggestion
4. Ensure **Monitor report messages in outlook** is selected and select **use the built-in report button**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTBen Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@MSFTBen Gentle reminder.

**Pull request merged**